api/src/db/migrations/v133_change_agent_office_address_city_field.ts

```diff
@@ -0,0 +1,494 @@
+import { v132Business, v132UserData } from "@db/migrations/v132_add_community_affairs_address";
+
+export interface v133ProfileData extends v133IndustrySpecificData {
+  businessPersona: v133BusinessPersona;
+  businessName: string;
+  responsibleOwnerName: string;
+  tradeName: string;
+  industryId: string | undefined;
+  legalStructureId: string | undefined;
+  municipality: v133Municipality | undefined;
+  dateOfFormation: string | undefined;
+  entityId: string | undefined;
+  employerId: string | undefined;
+  taxId: string | undefined;
+  encryptedTaxId: string | undefined;
+  notes: string;
+  documents: v133ProfileDocuments;
+  ownershipTypeIds: string[];
+  existingEmployees: string | undefined;
+  taxPin: string | undefined;
+  sectorId: string | undefined;
+  naicsCode: string;
+  foreignBusinessTypeIds: v133ForeignBusinessTypeId[];
+  nexusDbaName: string;
+  needsNexusDbaName: boolean;
+  operatingPhase: v133OperatingPhase;
+  isNonprofitOnboardingRadio: boolean;
+  nonEssentialRadioAnswers: Record<string, boolean | undefined>;
+  elevatorOwningBusiness: boolean | undefined;
+  communityAffairsAddress?: v133CommunityAffairsAddress;
+}
+
+export type v133CommunityAffairsAddress = {
+  streetAddress1: string;
+  streetAddress2?: string;
+  municipality: v133Municipality;
+};
+export const migrate_v132_to_v133 = (v132Data: v132UserData): v133UserData => {
+  return {
+    ...v132Data,
+    businesses: Object.fromEntries(
+      Object.values(v132Data.businesses)
+        .map((business: v132Business) => migrate_v132Business_to_v133Business(business))
+        .map((currBusiness: v133Business) => [currBusiness.id, currBusiness])
+    ),
+    version: 133,
+  } as v133UserData;
+};
+
+const migrate_v132Business_to_v133Business = (business: v132Business): v133Business => {
+  const agentOfficeAddressCityAlreadyExists =
+    business.formationData.formationFormData.agentOfficeAddressMunicipality?.displayName;
+
+  const v133BusinessObj = {
+    ...business,
+    formationData: {
+      ...business.formationData,
+      formationFormData: {
+        ...business.formationData.formationFormData,
+        agentOfficeAddressCity: agentOfficeAddressCityAlreadyExists || "",
```

Comment:
pebble - this just hit me during the re-review, the `agentOfficeAddressCity` key previously existed in user data. I think we need to consider the value that is stored within the key. Should the existing value be stored during the migration, it looks like we are overwriting it with the value stored in `agentOfficeAddressMunicipality` or an empty string. I don't think it was previously considered

Reply:
It would be helpful to get an understanding of the findings, perpahs a quick call once you have an idea on the best decision forword.